<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sendFileWindow</class>
 <widget class="QDialog" name="sendFileWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">发送文件</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>目标设备：</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea_2">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>378</width>
        <height>197</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="device_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">font: 75 15pt &quot;黑体&quot;;
color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>选择文件（拖放或者在资源管理器中添加）</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="file_scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="file_content_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>378</width>
        <height>197</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="files_table"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="remove_btn">
        <property name="text">
         <string>移除</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="select_file_btn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>选择文件 ...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancle_btn">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="send_btn">
        <property name="text">
         <string>发送</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
